ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -288,16 +288,6 @@ def _get_dtype_and_multiplicative_matrices(draw):
     )
 
 
-@st.composite
```

Comment:
Please do not remove any existing code!

Reply:
These lines prevents me from doing commits, the pre-commit tool catch those errors and block my commit.

Reply:
![image](https://github.com/unifyai/ivy/assets/31731234/65a5626d-d8db-47cf-8d2c-1f9c8b889aa2)

Reply:
Okay if this is the case, then please recommit without removing any code. This may be because of a failure from our side, but feel free to use the `--no-verify` flag to commit if everything else is okay, but do not remove existing code